docs/src/user_docs/assembly/io_operations.md

```diff
@@ -2,17 +2,17 @@
 Miden assembly provides a set of instructions for moving data between the stack and several other sources. These sources include:
 
 * **Program code**: values to be moved onto the stack can be hard-coded in a program's source code.
-* **Environment**: values can be moved onto the stack from environment variables. Currently, the only available environment variable is *stack_depth* which holds the current depth of the stack. In the future, other environment variables may be added.
+* **Environment**: values can be moved onto the stack from environment variables. Currently, the available environment variables are *stack_depth*, which stores the current stack depth, and *local_address*, which stores the absolute address of the local variable. In the future, other environment variables may be added.
```

Comment:
nit: I'd change the part about local addresses to: "... which stores absolute addresses of local variables."